ngraph/frontend/onnx_import/include/onnx_import/core/node.hpp

```diff
@@ -96,6 +98,114 @@ namespace ngraph
             std::unique_ptr<Impl, void (*)(Impl*)> m_pimpl;
         };
 
+        template <>
```

Comment:
Why do we need so much new exported methods?
What if user face with some type which is not covered by those methods?

Reply:
That's a  declaration of template specializations. We need to make those function externally available to avoid linker errors. Take a look here https://isocpp.org/wiki/faq/templates#separate-template-fn-defn-from-decl

Reply:
> What if user face with some type which is not covered by those methods?

then we'd need to extend the list also in node.cpp. But what we have, pretty much covers onnx spec: https://github.com/onnx/onnx/blob/master/onnx/onnx.proto#L117